app/views/podcast_episodes/_meta.html.erb

```diff
@@ -1,13 +1,13 @@
 <% if @podcast && @podcast.persisted? %>
-  <% title @podcast.title + " — The Practical Dev" %>
+  <% title @podcast.title %>
```

Comment:
`community_qualified_name` will be added inside the `title` method, so it doesn't make much sense to add it here.